includes/class-give-email-access.php

```diff
@@ -412,12 +412,19 @@ public function create_columns() {
 
 		global $wpdb;
 
-		// Create columns in customers table
-		$query = $wpdb->query( "ALTER TABLE {$wpdb->prefix}give_customers ADD `token` VARCHAR(255) CHARACTER SET utf8 NOT NULL, ADD `verify_key` VARCHAR(255) CHARACTER SET utf8 NOT NULL AFTER `token`, ADD `verify_throttle` DATETIME NOT NULL AFTER `verify_key`" );
+		$is_token_exists           = Give()->donors->is_column_exists( 'token' );
+		$is_verify_key_exists      = Give()->donors->is_column_exists( 'verify_key' );
+		$is_verify_throttle_exists = Give()->donors->is_column_exists( 'verify_throttle' );
 
-		// Columns added properly
-		if ( $query ) {
-			give_update_option( 'email_access_installed', 1 );
```

Comment:
@mehul0810 Write automatic upgrade to remove `email_access_installed`.

Reply:
@ravinderk Added Automatic Upgrade as well as improved the functioning of creating columns as discussed over call.